src/SFA.DAS.AANHub.Data/Repositories/MembersReadRepository.cs

```diff
@@ -32,38 +32,51 @@ internal class MembersReadRepository : IMembersReadRepository
 
     public async Task<List<MemberSummary>> GetMembers(GetMembersOptions options, CancellationToken cancellationToken)
     {
-        var regions = GenerateRegionsSql(options.RegionIds);
-        var userType = GenerateUserTypeSql(options.UserTypes, options.IsRegionalChair);
+        var regionsSql = GenerateRegionsSql(options.RegionIds);
+        var userTypeSql = GenerateUserTypeSql(options.UserTypes, options.IsRegionalChair);
         var keywordSql = GenerateKeywordSql(options.Keyword?.Trim());
+        var maturitySql = GenerateMaturitySql(options.IsNew);
 
         var sql = $@"SELECT Mem.[Id] AS MemberId
                       ,COUNT(*) OVER () TotalCount
                       ,Mem.[FullName]
-	                  ,Mem.[RegionId]
-	                  ,Reg.[Area] AS RegionName
-	                  ,Mem.[UserType]
-	                  ,Mem.[IsRegionalChair]
+                      ,Mem.[RegionId]
+                      ,Reg.[Area] AS RegionName
+                      ,Mem.[UserType]
+                      ,Mem.[IsRegionalChair]
                       ,Mem.[JoinedDate]
-                      FROM [Member] AS Mem
+                      ,(CASE WHEN DATEADD(day,90,[JoinedDate]) > GETUTCDATE() THEN CONVERT([BIT], (1)) ELSE CONVERT([BIT], (0)) END) AS IsNew
```

Comment:
this is comparing by date and time, I think we only want to compare by date, so this needs to be:
WHEN (CONVERT(date,DATEADD(day,90,[JoinedDate])) > CONVERT(date,GETUTCDATE()) THEN ....